src/Sushi.Mediakiwi/Controllers/Supporting/BaseController.cs

```diff
@@ -6,36 +6,7 @@
 
 namespace Sushi.Mediakiwi.Controllers
 {
-    public class BaseController : ControllerBase, IController
+    public class MediaKiwiController : ControllerBase, IController
```

Comment:
Class name mismatch with filename, please rename file to MediaKiwiController.cs
Also, is it Media**K**iwi or Media**k**iwi, since this is used interchangeably

Reply:
correct, resolved to Mediakiwi